### PR TITLE
Convert RUSDT to XUSD

### PR DESCRIPTION
**Pull request by tiltom**

### CODE REVIEW

File: src/app/components/UserAssets/ConversionDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            <AmountInput
+              value={weiToFixed(weiAmount, 6)}
+              onChange={noop}
+              asset={Asset.XUSD}
+              readonly={true}
+            />
+          </FormGroup>
+
+          <div className="tw-text-sm tw-font-thin tw-mt-8">
+            {t(translations.userAssets.convertDialog.txFee)}: 0.0006{' '}
```
Comment (grinry) [inline src/app/components/UserAssets/ConversionDialog.tsx:86]: Use TxFeeCalculator component to calculate real fee

File: src/app/hooks/portfolio/useConvertToXUSD.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  const rusdtAddress = getContract('USDT_token').address;
+  const { send, ...rest } = useSendContractTx('babelfishAggregator', 'mintTo');
+
+  return {
+    convert: async (weiAmount: string) => {
+      let tx: CheckAndApproveResult = {};
+
+      tx = await contractWriter.checkAndApprove(
+        Asset.USDT,
+        getContract('USDT_token').address,
```
Comment (grinry) [inline src/app/hooks/portfolio/useConvertToXUSD.ts:23]: Right now you are approving USDT token to be used by USDT token, spender must be babelfishAggregator here:

```suggestion
        getContract('babelfishAggregator').address,
```

**Review by grinry — CHANGES_REQUESTED** [review]

**Comment by creed-victor** [comment]
Do not merge to master until June 18th.

**Review by grinry — DISMISSED** [review]

**Review by grinry — DISMISSED** [review]

**Review by soulBit — DISMISSED** [review]

**Comment by t-z-u** [comment]
the launch of this has been delayed so could we please add the correct transaction window now it is not a rush to push

**Review by soulBit — APPROVED** [review]

**Review by creed-victor — APPROVED** [review]

**Pull request merged**